float64 leftMotorPower
float64 rightMotorPower
int32 leftMotorDirection
int32 rightMotorDirection
